src/axolotl/integrations/kernels/sonicmoe/routing.py

```
@@ -0,0 +1,219 @@
+    Args:
+        hidden_states: [T, H] flattened token representations
+        moe_block: MoE block module (accesses moe_block.gate.*)
+
+    Returns:
+        router_scores: [T*K] flattened scores (float32)
+        token_indices: [T*K] which token each entry belongs to (int32), sorted ascending
+        expert_indices: [T*K] which expert (int32)
+        router_logits: [T, E] original logits for aux loss
+    """
+    gate = moe_block.gate
+    T, H = hidden_states.shape
```

Ruff (0.15.2)

[warning] Unpacked variable `H` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)